python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -411,6 +433,70 @@ def readCatalogs(self, patchRefList, dataset):
         return concatenateCatalogs(catList)
 
     def correctForGalacticExtinction(self, catalog, tractInfo):
+        """Correct all fluxes for each object for Galactic Extinction
+
+        This function uses the EBVbase class from lsst.sims.catUtils.dust.EBV, so lsst.sims.catUtils must
+        be setup and accessible for use.
+
+        Parameters
+        ----------
+        catalog : `lsst.afw.table.SourceCatalog`
+           The source catalog for which to apply the per-object Galactic Extinction correction to all fluxes.
+           Catalog is corrected in place and a Galactic Extinction applied and flag columns are added.
+        tractInfo : `lsst.skymap.tractInfo.ExplicitTractInfo`
+           TractInfo object associated with catalog
+
+        Raises
+        ------
+        `ImportError`
+           If lsst.sims.catUtils.dust.EBV could not be imported.
+
+        Returns
+        -------
+        Updated `lsst.afw.table.source.source.SourceCatalog` catalog with fluxes corrected for
+        Galactic Extinction with a column added indicating correction applied and a flag indicating
+        if the correction failed (in the context having a non-np.isfinite value).
```

Comment:
in this context probably

Reply:
Actually it's supposed to be "in the context *of* having...".  Fixed.